composer.json

```diff
@@ -39,7 +39,7 @@
     "consolidation/filter-via-dot-access-data": "^1",
     "consolidation/output-formatters": "^3.3.1",
     "consolidation/robo": "^1.4.6",
-    "consolidation/site-alias": "^3.0.0-beta1",
+    "consolidation/site-alias": "dev-find-self-site as 3.0.0-beta1",
```

Comment:
clever ... change this before merge?

Reply:
Yes, I want feedback on this PR before I merge the site-alias PR, but I will merge and make a beta2 release before I merge here.